test/parallel/test-debug-port-cluster.js

```diff
@@ -3,47 +3,25 @@ var common = require('../common');
 var assert = require('assert');
 var spawn = require('child_process').spawn;
 
-var port = common.PORT + 1337;
+const PORT_MIN = common.PORT + 1337;
+const PORT_MAX = PORT_MIN + 2;
 
 var args = [
-  '--debug=' + port,
+  '--debug=' + PORT_MIN,
```

Comment:
I had that in my first (unpushed) revision but then I thought "meh, change for the sake of it and incongruous with the line below" so I left it out.